packages/shared/src/components/modals/badges/TopReaderBadgeModal.tsx

```diff
@@ -57,10 +58,11 @@ const TopReaderBadgeModal = (
         target_id: TargetId.TopReader,
         extra: JSON.stringify({
           tag: topReader.keyword.value,
+          origin,
```

Comment:
@idoshamun it gets added to all events for the modal, is that fine? Or do you just want it on the `impression` event?

Reply:
👍